wdlc.sh.in

```diff
@@ -88,6 +88,8 @@ function usage() {
         echo "                                       schema that can be used in place of "
         echo "                                       schema in a future invocation, or used "
         echo "                                       for version validation using the"
+        echo "  --output-dependency-file FILE        Emit dependency file information that is compatible"
```

Comment:
To improve first-approach accessibility for those familiar with other embedded tool chains, please include (a) compatible '-M' option(s) similar to gcc/clang for this, at minimum '-MF FILE'.

Reply:
Given that the dependency file we emit here does not look like a traditional make dependency file (it's missing the target), does it make sense to mimick their options despite the compatibility mismatch?

Reply:
What does it look like?

Reply:
A sample file:

```
 /usr/local/include/google/protobuf/descriptor.proto\
 /usr/local/include/google/protobuf/any.proto\
 /usr/local/include/google/protobuf/duration.proto\
 /usr/local/include/google/protobuf/timestamp.proto\
 /Users/johnsj/work/devel/git/gob/phoenix-schema/openweave-wdlc/build/results/bin/wdlc-1.0/../../include/wdlc-1.0/storage/datapol/annotations/proto/retention_annotations.proto\
 /Users/johnsj/work/devel/git/gob/phoenix-schema/openweave-wdlc/build/results/bin/wdlc-1.0/../../include/wdlc-1.0/storage/datapol/annotations/proto/semantic_annotations.proto\
 /Users/johnsj/work/devel/git/gob/phoenix-schema/openweave-wdlc/build/results/bin/wdlc-1.0/../../include/wdlc-1.0/wdl/data_access.proto\
 /Users/johnsj/work/devel/git/gob/phoenix-schema/openweave-wdlc/build/results/bin/wdlc-1.0/../../include/wdlc-1.0/wdl/vendors.proto\
```

Reply:
If that's the output, we should output that to a temporary file and process it to something specified by the user that actually works directly with make the way that `gcc -M*` and friends work.

Reply:
We don't have a really good implicit option for a target here, so if we went the make-option route, we'd have to require a `-MT` option in addition to the `-MF` option as well.